lib/bitcoin/BitcoinClient.ts

```diff
@@ -150,12 +150,12 @@ export default class BitcoinClient {
    * this transaction before spending from the wallet.
    *
    * @param lockAmountInSatoshis The amount to lock.
-   * @param lockUntilBlock  The block until the amount to lock to; the amount becomes spendable AT this block.
+   * @param lockDurationInBlocks  The number of blocks to lock the amount for; the amount becomes spendable AFTER this many blocks.

```

Comment:
If the lock is written to the chain at block 100, and we set the lock duration to 5, it will unlock at 105? (100, 101, 102, 103, 104) Just for my own knowledge more than anything :p

Reply:
That is correct @isaacJChen

Reply:
Yes is my understanding.